torch/_inductor/codegen/cuda/cutlass_utils.py

```diff
@@ -0,0 +1,170 @@
+import functools
+import logging
+from typing import Any, List
+
+# Import cutlass python scripts.
+import generator as cutlass_generator  # type: ignore[import]
+import library as cutlass_lib  # type: ignore[import]
+import manifest as cutlass_manifest  # type: ignore[import]
```

Comment:
These are very generic names that could be used in user code.  Given how many users of PyTorch there are I am sure someone has a module named `library` in some closed source project.

We should put these in some sort of `cutlass_foo.*` namespace.